composer.json

```diff
@@ -27,6 +27,7 @@
         "orchestra/testbench": "^6.0",
         "phpunit/phpunit": "^9.3",
         "spatie/laravel-ray": "^1.9",
+        "tipoff/test-support": "dev-pdbreen/bugfix/remove-authorization",
```

Comment:
@drewroberts - need a version bump here (flagging since it wasn't part of my original comment).

Reply:
We had to remove it before since tipoff/test-support requires this tipoff/authorization package and we couldn't have circular dependency

Reply:
test-support should not include authorization.  I've removed it.  But this package **MUST** have test-support.  None of these tests will run without it.